app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -116,8 +156,14 @@ ProjectBoard.propTypes = {
   fetchProjectBoard: PropTypes.func.isRequired,
   createStory: PropTypes.func.isRequired,
   closeHistory: PropTypes.func.isRequired,
-  notifications: PropTypes.array.isRequired
-}
+  notifications: PropTypes.array.isRequired,
+  reverseColumns: PropTypes.func.isRequired,
+  projectId: PropTypes.string.isRequired,
+  removeNotification: PropTypes.func.isRequired,
+  history: PropTypes.object.isRequired,
```

Comment:
Not sure if you do it in the other PR, but it'd be nice to have a proper history shape to use here.

Reply:
Oh, for sure. I will do this in PR of history.